### DISCUSSION TITLE
Data Importer not displaying properly

### DISCUSSION THREAD

**Discussion by Horselsama** [firefly-iii/firefly-iii#6837] [Firefly III]
Hello,
I've been having this problem with the Data Importer in which the website does not display properly. I'm using docker as my instalation and running through ngnix-proxy. I have attached the logs and a screnshot of how the webiste looks. 

![image](https://user-images.githubusercontent.com/107134267/212055656-06d6cb28-9a6b-433d-9079-eaf34c0bc24e.png)
[_FireflyDataImporter_logs.txt](https://github.com/firefly-iii/firefly-iii/files/10401140/_FireflyDataImporter_logs.txt)

As you can see the buttons does not render properlly maiking it not possible to upload the files.

**Comment by JC5** [comment]
Check your reverse proxy configuration to verify it sends the host and port etc.
